app/src/test/java/com/codesoom/assignment/application/AccountServiceTest.java

```diff
@@ -0,0 +1,129 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.AccountNotFoundException;
+import com.codesoom.assignment.domain.Account;
+import com.codesoom.assignment.domain.AccountRepository;
+import com.github.dozermapper.core.DozerBeanMapperBuilder;
+import com.github.dozermapper.core.Mapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+class AccountServiceTest {
+    private AccountService accountService;
+    private AccountRepository accountRepository = mock(AccountRepository.class);
+    private Account account;
+
+    private final static Long REGISTERED_ID = 1L;
+    private final static Long NOT_REGISTERED_ID = 100L;
+
+    @BeforeEach
+    void setUp() {
+        Mapper mapper = DozerBeanMapperBuilder.buildDefault();
+        accountService = new AccountService(accountRepository, mapper);
+
+        account = Account.builder()
+                .id(1L)
+                .name("Tester")
+                .password("password")
+                .email("email@email.com")
+                .build();
+    }
+
+    @Nested
+    @DisplayName("getAccounts 메서드는")
+    class Describe_getAccounts {
+
+        @Nested
+        @DisplayName("회원정보가 존재하면")
+        class Context_with_exist_account {
+            @BeforeEach
+            void setUp() {
+
+                given(accountRepository.findAll())
+                        .willReturn(List.of(account));
+            }
+
+            @Test
+            @DisplayName("전체 회원정보 목록을 리턴합니다.")
+            void it_return_account_list() {
+                assertThat(accountService.getAccounts())
+                        .isNotEmpty();
+
+                verify(accountRepository).findAll();
+            }
+        }
+
+        @Nested
+        @DisplayName("회원정보가 존재하지 않으면")
```

Comment:
이렇게 하는 것이 더 낫겠습니다.
```suggestion
        @DisplayName("회원정보를 찾을 수 없다면")
```

왜냐하면, 준비 코드에서 `존재`를 준비하는 것이 아니라 `find`를 준비하고 있기 때문이에요.
```
            void setUp() {
                given(accountRepository.findAll())
                        .willReturn(List.of());
            }
```